docker/docker_client.go

```diff
@@ -277,9 +279,15 @@ func newDockerClient(sys *types.SystemContext, registry, reference string) (*doc
 	}
 	tlsClientConfig.InsecureSkipVerify = skipVerify
 
+	userAgent := "containers-image/" + version.Version
```

Comment:
Any chance we can do something like "github.com/containers/image"?  Not sure that's allowed but the source would be more explicit.

Reply:
> Any chance we can do something like "github.com/containers/image"? Not sure that's allowed but the source would be more explicit.


I believe "something like", yes, but not exactly:

```
       User-Agent     = "User-Agent" ":" 1*( product | comment )

       product         = token ["/" product-version]

       product-version = token

       token          = 1*<any CHAR except CTLs or tspecials>

       tspecials      = "(" | ")" | "<" | ">" | "@"
                      | "," | ";" | ":" | "\" | <">
                      | "/" | "[" | "]" | "?" | "="
                      | "{" | "}" | SP | HT

       comment        = "(" *( ctext | comment ) ")"

       ctext          = <any TEXT excluding "(" and ")">

       TEXT           = <any OCTET except CTLs,
                        but including LWS>
```

From https://tools.ietf.org/html/rfc1945#section-2.2 and https://tools.ietf.org/html/rfc1945#section-3.7 and https://tools.ietf.org/html/rfc1945#section-10.15

`token` forbids `/`, but we could add it to the `comment`, which can include anything other than `(`, `)`, control characters, and newlines.

How about this?

```
User-Agent: containers/$VERSION (github.com/containers/image)
```

Reply:
Thanks for looking into it. I really like it :)

@rhatdan @mtrmac PTAL